Sources/Protocols/DataLoading/DataSource.swift

```diff
@@ -21,7 +21,7 @@
 //
 
 /// Protocol for data sources that has type of loaded result
-public protocol DataSourceProtocol {
+public protocol DataSource {
```

Comment:
What for this protocol?

Reply:
In theory - for non-rx DataSources. To know result type of data that it provides. Can be merged with RxDataSource.